### ISSUE TITLE
RGB Color results to NaN in production with terser webpack plugin

### ISSUE THREAD

**Issue by JSteitz** [Simonwep/pickr#76]
When I use this package as an es6 import `import Pickr from "@simonwep/pickr"` and build my assets with `terser-webpack-plugin` for production, I always get a NaN for RGB values.
If I use the source code as import (`import Pickr from "@simonwep/pickr/src/js/pickr"`) then I don't get any problems. The same applies when I don't use the terser plugin.

If you don't think this is a bug in pickr (that's what I think), then can you add this as "Known Issues" or "Gotcha's" in the Readme for other people.

**Comment by Simonwep** [comment]
It seems like somehow the `main` entry in `package.json` is wrong... Anyway, your issue is documented under [Setup - Node](https://github.com/Simonwep/pickr#node) but yeah, it _should_ work with `import Pickr from "@simonwep/pickr"` since it points to the minified version... I'll take a look!

**Comment by Simonwep** [comment]
Okay I've tried to reproduce it with this setup:

<details>
<summary>app.js</summary>

```js
import '@simonwep/pickr/dist/pickr.min.css';
import Pickr from '@simonwep/pickr/dist/pickr.min';

const pickr = Pickr.create({
    el: '.color-picker',

    components: {

        // Main components
        preview: true,
        opacity: true,
        hue: true,

        // Input / output Options
        interaction: {
            hex: true,
            rgba: true,
            hsla: true,
            hsva: true,
            cmyk: true,
            input: true,
            clear: true,
            save: true
        }
    }
});
```
</details>

<details>
<summary>webpack.config.js</summary>

```js
const TerserPlugin = require('terser-webpack-plugin');
const webpack = require('webpack');

module.exports = {
    entry: './src/app.js',

    output: {
        path: __dirname + '/dist',
        publicPath: 'dist/',
        filename: 'bundle.min.js'
    },

    devServer: {
        contentBase: __dirname + '/',
        host: '0.0.0.0',
        port: 3002
    },

    module: {
        rules: [
            {
                test: /\.js$/,
                use: 'babel-loader'
            },
            {
                test: /\.css$/,
                use: ['style-loader', 'css-loader']
            }
        ]
    },

    plugins: [
        new webpack.SourceMapDevToolPlugin({
            filename: 'bundle.js.map'
        })
    ],

    optimization: {
        minimizer: [new TerserPlugin()]
    }
};
```
</details>

But everything seems fine :suspect:

**Comment by JSteitz** [comment]
Sorry for the late reply.

I have some more problems with the current version.
When I identify the problems and sort them out I will make a working code examples with the bug.

**Comment by JSteitz** [comment]
Whatever you did between version 4.4 and 4.7 fixed my problem.
But the position is now off :/

**Comment by Simonwep** [comment]
Glad to hear it got fixed by itself :)
What do you mean with "position is now off"

**Comment by JSteitz** [comment]
The position for the popup is not calculated correctly, it is always top: 0 left: 0 with button height as offset.
Will need to create a ticket later.

**Comment by Simonwep** [comment]
Yeah, it get's positioned only with `margin`. See #79 and where it got fixed: ab8b3c7bf5d573bd49358f5a0726b7dc78ad424d

And useAsButton is broken.

**Comment by JSteitz** [comment]
Sadly I need `useAsButton`, but it is not a deal breaker for now  :)

**Comment by Simonwep** [comment]
@JSteitz I've published (for now) 0.4.8 where positioning related stuff _should be_ more stable :suspect:. I've removed the parent option therefore - pickr is now always appended to the body. I'll consider giving an option to add a class or similar to apply custom styles.
